### PR TITLE
Fix bug bounds bigger than long can take

### PR DESCRIPTION
**Pull request by ricketytoc**
Handles the case where if the user inputs a bound larger than a long can take then a NumberFormatException will be thrown

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 1]
```diff
@@ -329,9 +329,14 @@ private static boolean isValidFindSalaryRange(String salaryRange) {
...
-        long lowerBoundLong = SalaryParserUtil.parseStringToLong(lowerBound);
-        if (upperBoundLong > Salary.MAXIMUM_SALARY_LONG || lowerBoundLong > upperBoundLong) {
+        try {
+            long upperBoundLong = SalaryParserUtil.parseStringToLong(upperBound);
+            long lowerBoundLong = SalaryParserUtil.parseStringToLong(lowerBound);
+            if (upperBoundLong > Salary.MAXIMUM_SALARY_LONG || lowerBoundLong > upperBoundLong) {
+                return false;
+            }
+        }
+        catch (NumberFormatException e) {
```
Comment (Goh-Li-Ting) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java:339]: Should we modify `SalaryParserUtil#parseStringToLong` to catch `NumberFormatException` and then throw `ParseException` instead? I just realised that the current way of handling `NumberFormatException` might have revealed the implementation of `SalaryParserUtil#parseStringToLong`

Comment (Goh-Li-Ting) [reply src/main/java/seedu/address/logic/parser/ParserUtil.java:339]: Will fix in a later pr

File: src/test/java/seedu/address/logic/parser/FindCommandParserTest.java [hunk 2]
```diff
@@ -212,22 +221,22 @@ public void parse_validArgs_returnsFindCommand() {
...
+        // (BVA) Salary attribute: lower bound and upper bound are both max salary
         predicateList.clear();
         predicateList.add(new SalaryWithinRangePredicate(Salary.MAXIMUM_SALARY_LONG, Salary.MAXIMUM_SALARY_LONG));
         expectedFindCommand =
                 new FindCommand(new GeneralPredicate(predicateList));
         assertParseSuccess(parser, " " + PREFIX_SALARY + Salary.MAXIMUM_SALARY + " - " + Salary.MAXIMUM_SALARY,
                 expectedFindCommand);
-
-        // Salary attribute - Boundary Value Analysis: lower and upper bound are both zero
+
```
Comment (Chen-Kuei) [inline src/test/java/seedu/address/logic/parser/FindCommandParserTest.java:231]: should not have this extra white space at line 231

**Review by Weiennn — APPROVED** [review]
LGTM

**Review by Goh-Li-Ting — APPROVED** [review]
LGTM
